plutus-tx-plugin/src/PlutusTx/Compiler/Builtins.hs

```diff
@@ -187,7 +188,7 @@ builtinNames = [
     , 'Builtins.error
 
     , ''Builtins.BuiltinString
-    , ''Char
+    , ''Text
```

Comment:
Shouldn't need this, `BuiltinString` should map to `Text`, not `Text` itself.

Reply:
```suggestion

```